chart/templates/iq-server-deployment.yaml

```diff
@@ -193,6 +193,10 @@ spec:
                   {{- end }}
         {{- if and (.Values.fluentd.enabled) (.Values.iq_server.fluentd.forwarder.enabled) }}
         - name: {{ .Release.Name }}-fluentd-container
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop: [ "ALL" ]
```

Comment:
I'm not sure how this would affect flux, probably need to manually test that it still works

Reply:
Flux shouldn't be operating inside the pods context

Reply:
Sorry editted i meant `fluentd` not `flux`

Reply:
I changed this to be more backwards compatible, allowing it to be configured by users, but defaults to empty as before.